client/src/components/ElementActions/PublishAction.js

```diff
@@ -9,21 +9,20 @@ import i18n from 'i18n';
  * Adds the elemental menu action to publish a draft/modified block
  */
 const PublishAction = (MenuComponent) => (props) => {
+  const { element, actions: { handlePublishBlock } } = props;
```

Comment:
I think `element` here could become `element: { id, Title: title, BlockSchema, Version, isLiveVersion }` and remove a lot of the other changes (effectively unnecessary repetition).
Same with the other `Actions` HoCs.

Reply:
I considered this but thought it was being sort of over-verbose:

`element: { ID: id, Title: title, BlockSchema: { type }, Version: version, IsLiveVersion: isLiveVersion }`

We've not de-structured `element` in some other places too.

I will change it though - it doesn't really bother me what we end up doing.

Reply:
Yeah, tbh I was of 2 minds about it. Having the prefix makes it obvious where the value comes from too.
But I think it's that we prefer destructuring where possible.

Reply:
On second thought, I don't care that strongly about this.